Automattic/simplenote-ios
# Add Append note Intent

### Fix
So continuing on with the shortcut improvements.  One of the requested shortcuts is to be able to append to an existing note.  Makes sense that you would want to be able to use shortcuts to easily add to one of your existing notes.  So in this PR I have done just that.  

This PR adds a new intent Append Note.  With Append note you can specify an existing note and content that you want to add to it.  You can also choose to select the note and the content at the time which makes this shortcut a pretty versatile way to add content to existing Simplenote notes from outside of the app.

### Test
1. launch Simplenote with this branch and log into an account (you may need to do this on device to make the new entitlements work)
2. background Simplenote and go to the shortcuts app.  Add a new shortcut, add action and look for Simplenote.  Select Append note
3. Tap on the content button and choose "ask every time" and then tap on the note and select a note to use
4. Run the shortcut, add some content to your note.
5. Open Simplenote again and confirm that your note has been updated with the new content.

A few things to consider while testing this:
- make sure that tags stay the same
- make sure that system tags stay the same
- make sure a published note continues to be published

### Review
***(Required)*** Add instructions for reviewers.  For example:
> Only one developer is required to review these changes, but anyone can perform the review.

### Release
***(Required)*** Add a concise statement to `RELEASE-NOTES.txt` if the changes should be included in release notes. Include details about updating the notes in this section. For example:
> These changes do not require release notes.

(When the last of the shortcuts are added then I will update the release notes)
